### PR TITLE
Update appsettings.Development.json

### LINKED ISSUE #29512: Directions out of order
1. To see the final output as shown, one must do this BEFOREHAND (not afterwards):
Update the appsettings.Development.json file by adding the following lines:
"Microsoft.AspNetCore.Hosting": "Information",
"Microsoft.AspNetCore.Routing.EndpointMiddleware": "Information"
2. An additional comma is needed before these added lines in appsettings.Development.json

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1d298a6c-8ddc-dbc8-d219-0fa7138442c2
* Version Independent ID: b96be346-958a-cfcd-d80a-c352d64223e8
* Content: [Create a .NET Core gRPC client and server in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-7.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/grpc/grpc-start.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/grpc/grpc-start.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @JamesNK
* Microsoft Alias: **jamesnk**

### PR DESCRIPTION
**Pull request by postmeback**
Added text as per given instruction in #29512

Fixes #29512

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/grpc/grpc-start.md](https://github.com/dotnet/AspNetCore.Docs/blob/b26553d43c5c30dfa1ae3140872ff1746dfd3c75/aspnetcore/tutorials/grpc/grpc-start.md) | [Tutorial: Create a gRPC client and server in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?branch=pr-en-us-29549) |

### CODE REVIEW

File: aspnetcore/tutorials/grpc/grpc-start/sample/sample8/GrpcGreeter/appsettings.Development.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
   "Logging": {
     "LogLevel": {
       "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
+      "Microsoft.AspNetCore": "Warning",
+      "Microsoft.AspNetCore.Hosting": "Information",
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/grpc/grpc-start/sample/sample8/GrpcGreeter/appsettings.Development.json:6]:
```suggestion
      "Microsoft.AspNetCore": "Warning"
      ,"Microsoft.AspNetCore.Hosting": "Information",
```

Comment (Rick-Anderson) [reply aspnetcore/tutorials/grpc/grpc-start/sample/sample8/GrpcGreeter/appsettings.Development.json:6]: Do it like this to make the instructions simpler.

**Comment by postmeback** [comment]
This PR is updated with the necessary file changes. Requesting for review

File: aspnetcore/tutorials/grpc/grpc-start.md [hunk 2]
```diff
@@ -287,6 +287,10 @@ For Visual Studio for Mac, see the .NET 7 version of this tutorial.
 
 ---
 
+Update the `appsettings.Development.json` file by adding the following highlighted lines:
+
+[!code-csharp[](~/tutorials/grpc/grpc-start/sample/sample8/GrpcGreeter/appsettings.Development.json?highlight=6-7)])
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/grpc/grpc-start.md:292]: [!code-csharp[](~/tutorials/grpc/grpc-start/sample/sample8/GrpcGreeter/appsettings.Development.json?highlight=6-7)]

Comment (Rick-Anderson) [inline aspnetcore/tutorials/grpc/grpc-start.md:292]:
```suggestion
[!code-csharp[](~/tutorials/grpc/grpc-start/sample/sample8/GrpcGreeter/appsettings.Development.json?highlight=6-7)]
```

**Comment by wadepickett** [comment]
Testing through this in a moment...

**Review by wadepickett — APPROVED** [review]
Looks good, thanks!

**Pull request merged**